### ISSUE TITLE
Multiple errors when running the configure script

### ISSUE THREAD

**Issue by jamesnhan** [T-Grave/goxlr-on-linux#8]
The configure_goxlr.sh script produces the following errors:

```
--- engine param set "realtime" -> "true"
--- driver select "alsa"
--- driver param set "device" -> "hw:GoXLR,0"
--- driver param set "period" -> "512"
--- driver param set "rate" -> "48000"
--- driver param set "nperiods" -> "3"
--- start
Failure: Module initialization failed
Failure: Module initialization failed
Failure: Module initialization failed
Failure: Module initialization failed
Failure: Module initialization failed
Failure: Module initialization failed
Failure: Module initialization failed
Cannot lock down 107341338 byte memory area (Cannot allocate memory)
cannot connect client, already connected?
Cannot lock down 107341338 byte memory area (Cannot allocate memory)
cannot connect client, already connected?
Cannot lock down 107341338 byte memory area (Cannot allocate memory)
cannot connect client, already connected?
Cannot lock down 107341338 byte memory area (Cannot allocate memory)
cannot connect client, already connected?
Cannot lock down 107341338 byte memory area (Cannot allocate memory)
cannot connect client, already connected?
Cannot lock down 107341338 byte memory area (Cannot allocate memory)
cannot connect client, already connected?
Cannot lock down 107341338 byte memory area (Cannot allocate memory)
cannot connect client, already connected?
Cannot lock down 107341338 byte memory area (Cannot allocate memory)
cannot connect client, already connected?
Cannot lock down 107341338 byte memory area (Cannot allocate memory)
ERROR GoXLR_Sink_Sample:front-left not a valid port
Cannot lock down 107341338 byte memory area (Cannot allocate memory)
ERROR GoXLR_Sink_Sample:front-right not a valid port
Cannot lock down 107341338 byte memory area (Cannot allocate memory)
ERROR GoXLR_Source_Broadcast_Mix:front-left not a valid port
Cannot lock down 107341338 byte memory area (Cannot allocate memory)
ERROR GoXLR_Source_Broadcast_Mix:front-right not a valid port
Cannot lock down 107341338 byte memory area (Cannot allocate memory)
ERROR GoXLR_Source_Chat:front-left not a valid port
Cannot lock down 107341338 byte memory area (Cannot allocate memory)
ERROR GoXLR_Source_Chat:front-right not a valid port
```

Possibly related to the missing audio.config issue: #7

**Comment by T-Grave** [comment]
Could def. be the case.

Also make sure you have added your user to the audio group as described in this known issue:
https://github.com/T-Grave/goxlr-on-linux#cannot-lock-down-107341338-byte-memory-area-cannot-allocate-memory

**Comment by T-Grave** [comment]
@jamesnhan PR #9 has been merged so you should be able to find the file and try again. Let us know if it worked!

**Comment by jamesnhan** [comment]
The issue was resolved after #9 . The issue with the invalid ports was also due to #7, which can be misleading. My user was already in the `audio` group, but I still got that error.

That being said, #9 fixed all issues so far!
